blog/2021-12-17-invisible-characters-for-better-localization.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using invisible text for better localization
+authors: [sgranat]
+tags: [SDK, i18n, i18next]
+---
+
+![Text with ghosts](/img/blog/invisible-text/invisible_text.png)
+
+Some time ago I've came across an [article](https://null-byte.wonderhowto.com/how-to/use-zero-width-characters-hide-secret-messages-text-even-reveal-leaks-0198692/) about inserting secret text into regular messages using invisible zero-width unicode characters. Now during my work on localization SDKs, I've found quite interesting usecase for it.
+
+<!--truncate-->
+
+## Background
+
+At tolgee we are offering localization SDKs, which enable you insert your translated keys into web application. One of our main selling points is that we offer in-context localization out of box. In-context localization means, that when you are in development mode, you can click on any translation in the app and change the translation directly in the app.
```

Comment:
Tolgee

Reply:
"which enable you +to"?

in the app too many times?
on any translation -`in the app` and change the translation directly in the app.